client/my-sites/customer-home/free-photo-library-card/index.jsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { localize } from 'i18n-calypso';
+import { Card, Button, Dialog } from '@automattic/components';
+import { isDesktop } from '@automattic/viewport';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'components/card-heading';
+import { localizeUrl } from 'lib/i18n-utils';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+/**
+ * Image dependencies
+ */
+import freePhotoLibraryThumbnail from 'assets/images/customer-home/free-photo-library-thumbnail.png';
+import freePhotoLibraryVideoPrompt from 'assets/images/customer-home/free-photo-library-video-prompt.png';
+
+class FreePhotoLibraryCard extends Component {
+	state = {
+		showDialog: false,
+	};
```

Comment:
Optional. This component is quite simple, so we could keep it as a functional component with hooks ([as recommended by React](https://reactjs.org/docs/hooks-intro.html#gradual-adoption-strategy)):

```js
import React, { useState } from 'react';
import { useTranslate } from 'i18n-calypso';

const FreePhotoLibraryCard = () => {
  const [ showDialog, setShowDialog ] = useState( false );
  const translate = useTranslate();

  const toggleDialog = () => setShowDialog( ! showDialog );
  return (
    // ...
  );
}
```

Reply:
Interesting, thanks! I've updated the PR to use that approach. :)